frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature

```diff
@@ -0,0 +1,95 @@
+@add-flow
+Feature: Add page on Developer Console
+              As a user, I should be able select a way to create an Application, component or service from one of the options provided on Add page
+
+    
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-add-page"
+              And user has installed OpenShift Serverless Operator
+              And user has installed OpenShift Pipelines operator
+              And user is at Add page
+
+        
+        @regression
+        Scenario: Getting started resources on Developer perspective
+             Then user will see Getting started resources
+              And user will see Create Application using Samples
+              And user will see Build with guided documentation
+              And user will see Explore new developer features
+
+
+        @smoke
+        Scenario: Options to create an Application, Component and Service
+             Then user will see "Developer Catalog" card
+              And user will see "Git Repository" card
+              And user will see "Container Images" option
```

Comment:
nit:
"Container images" to be consistent with how it appears in the UI.

Reply:
Updated